gui/_menu.scss

```diff
@@ -132,6 +139,11 @@ ul {
       transform: translateY(-50%);
       pointer-events: none;
     }
+
+    i {
+      font-style: normal;
```

Comment:
This isn't semantically appropriate, i.e. overriding the default `font-style` of the native `<i>` tag.